tests/testthat/test_stan_jm.R

```diff
@@ -520,6 +520,17 @@ for (j in c(1:30)) {
       }  
     }) 
     
+    test_that("posterior_traj works with new data with and without offset", {
+      nd <- pbcLong[1:2, ]
```

Comment:
Same here I think, we might not want the response in the new data, so that we can catch the original error, something like:

```
nd <- pbcLong2[1:2, c("id", "year")]
```